parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -383,7 +383,7 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     {{else}}
     PRIVATE_IP=$(hostname -i | cut -d" " -f1)
     {{end}}
-    ETCD_CLIENT_PORT={{WrapAsVariable "masterEtcdClientPort"}}
+    sed -i "s|<SERVERIP>|https://$PRIVATE_IP:443|g" "/var/lib/kubelet/kubeconfig"
```

Comment:
Just to make sure I understand, do we only need to do this for vmss? Is this field not present in /var/lib/kubelet/kubeconfig for vmas?

It seems like we were doing this sed statement for both vmss and vmas before but now only for vmss.

Reply:
Looks like that field isn't present for vmas